debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -374,18 +385,28 @@ protected void handleServerHeartbeat(Event event) {
     /**
      * Handle the supplied event that signals that an out of the ordinary event that occurred on the master. It notifies the slave
      * that something happened on the master that might cause data to be in an inconsistent state.
-     * 
+     *
      * @param event the server stopped event to be processed; may not be null
      */
     protected void handleServerIncident(Event event) {
-        logger.trace("Server incident: {}", event);
+        if (event.getData() instanceof EventDataDeserializationExceptionData) {
+            EventDataDeserializationExceptionData data = event.getData();
+
+            logger.error(
+                    String.format("Exception during event data deserialization (source info: %s, partion: %s)", source, source.partition()),
```

Comment:
Why are you using `String.format` here? The `error(...)` method takes parameters like `{}`. So in this case, you'd do something like:
```
  logger.error("Error while deserializing binlog event at {}", source.offset(), data.getCause());
```
Note how the exception is 1 more argument than the last parameter value.

But as I suggest in my next comment, it'd be good to also mention in this log message how to view the event with `mysqlbin`, so:
```
  logger.error("Error while deserializing binlog event at {}\nUse the `mysqlbinlog` tool to view the problematic event with `mysqlbinlog --start-position={} --stop-position={} --verbose {}", 
           source.offset(), source.binlogPosition(), source.nextBinlogPosition(), source.binlogFilename(), data.getCause());
```

Reply:
Done.